toeverything/AFFiNE
# feat: added duplicate page button in page list and clone naming improvements

# Enhancement: Improved Duplicate Page Functionality

## Changes Made:

## Added Separate Duplicate Button:

The duplicate page option was initially confined within the current active pages header. To enhance user experience, a separate button has been introduced in the `PageOperationCell` for easier accessibility.
Repositioning Duplicate Option:

The new duplicate button is strategically placed above the existing MoveToTrash option in the PageOperationCell for better visibility and logical organization of page operations.
Updated Duplicate Function:

Modified the duplicate function to automatically increment the suffix number of the duplicated page. This improvement aligns with standard editing tool behavior.
How to Test:

Click on the new duplicate button in the `PageOperationCell`.
Confirm that the duplicate page option is now easily accessible.
Duplicate a page and observe that the suffix number increments appropriately.
Note:
This enhancement aims to streamline the process of duplicating pages and ensure a more intuitive user interface. The incremented suffix number for duplicated pages adds clarity and aligns with user expectations.
